### DISCUSSION TITLE
How do you organize labels

### DISCUSSION THREAD

**Discussion by N6REJ** [community/community#29465] [New to GitHub] [answered]
How can labels be sorted?
I have an action that sync's labels across all repos but they seem to be in some random order

**Comment by luckylittle** [comment]
Hi @N6REJ 

They are alphabetical. Look at this example:

```bash
curl \
  -H "Accept: application/vnd.github+json" \
  -H "Authorization: token <OMITTED>" \
  https://api.github.com/repos/luckylittle/openshift-cheatsheet/labels
```

It returns the default ones:

```json
[
  {
    "name": "bug",
  },
  {
    "name": "duplicate",
  },
  {
    "name": "enhancement",
  },
  {
    "name": "help wanted",
  },
  {
    "name": "invalid",
  },
  {
    "name": "question",
  },
  {
    "name": "wontfix",
  }
]
```

When I add a new one called `new_label` and run it again, it is still alphabetical:

```json
[
  {
    "name": "bug",
  },
  {
    "name": "duplicate",
  },
  {
    "name": "enhancement",
  },
  {
    "name": "help wanted",
  },
  {
    "name": "invalid",
  },
  {
    "name": "new_label",   <--- here is the new one
  },
  {
    "name": "question",
  },
  {
    "name": "wontfix",
  }
]
```

_Note: Please accept the answer if it answers your question, thanks_

**Reply by queenofcorgis** [reply]
Thanks for chiming in @luckylittle - appreciate your perspective!

**Comment by N6REJ** [comment]
Thats whats got me confused...
<img width="275" alt="image" src="https://user-images.githubusercontent.com/1850089/184709665-91e8a346-de19-4eef-baea-912b8fc064f5.png">
Clearly this is not alphabetical

**Reply by queenofcorgis** [reply]
Could the emojis be throwing off the alphabetical nature?

**Reply by N6REJ** [reply]
no clue

**Comment by luckylittle** [comment, marked as answer]
@N6REJ The labels are alphabetical when you consider the emojis - look at this:

```
":arrow_forward: Sync"
":books: documentation"
":bug: bug"
":bulb: feature request"
":busts_in_silhouette: duplicate"
":closed_lock_with_key: Security"
...
...
```

My suggestion is to put the emojis after the keywords

_Note: Please accept the answer if it answers your question, thanks_

**Reply by N6REJ** [reply]
yes that fixed it!  tyvm!

**Reply by queenofcorgis** [reply]
Thanks @luckylittle for diving deeper 🤿  and figuring this out! Appreciate you marking the solution, @N6REJ :octocat:
